src/modules/account/const/my-history.const.tsx

```diff
@@ -80,6 +81,17 @@ export const HistoryTableColumns = (
 	{
 		Header: '',
 		id: 'account-history-placeholder',
-		Cell: () => null,
+		Cell: (data: VisitRow) => {
+			const visit = data.row.original;
+			return (
+				<Button
+					className="u-font-size-14 u-py-16 u-text-left"
+					variants={['text', 'block', 'fill']}
+					onClick={() => onClickRow(visit)}
+				>
+					{i18n.t('modules/account/const/my-history___bezoek')}
+				</Button>
```

Comment:
should this button be just text or should it be a green button? Currently i just see text:

![image](https://user-images.githubusercontent.com/1710840/166212266-b88e8408-507a-4623-9f0b-8caee894edeb.png)

Reply:
Styling moet volgens Christophe hetzelfde zijn zoals bij de aanvragen actions. (zonder badge dan)
![image](https://user-images.githubusercontent.com/85545258/166435318-b23435db-c90f-4fec-ad7f-278923ee159a.png)

Reply:
maar dan ook in het vet?

Reply:
@bertyhell 'Bezoek' staat niet vet, maar is gewoon zwart.
@ian-emsens-shd Checked. Huidige styling blijft.